launch/vehicle_interface.launch.xml

```diff
@@ -2,11 +2,15 @@
 <launch>
 
   <arg name="vehicle_id" default="$(env VEHICLE_ID default)"/>
-
+  <arg name="csv_accel_brake_map_path" default="$(find-pkg-share individual_params)/config/$(var vehicle_id)/vehicle"/>
   <arg name="max_throttle" default="0.4"/>
   <arg name="max_brake" default="0.8"/>
+  <arg name="convert_accel_cmd" default="true"/>
+  <arg name="convert_brake_cmd" default="true"/>
+  <arg name="convert_steer_cmd" default="false"/>
 
-  <arg name="pacmod_param_path" default="$(find-pkg-share pacmod_interface)/config/pacmod.param.yaml"/>
+  <arg name="pacmod_param_path" default="$(find-pkg-share individual_params)/config/$(var vehicle_id)/vehicle/pacmod.param.yaml"/>
+  <arg name="pacmod_extra_param_path" default="$(find-pkg-share individual_params)/config/$(var vehicle_id)/vehicle/pacmod_extra.param.yaml"/>
```

Comment:
The directory structure of parameter files are different from that of japantaxi

https://github.com/tier4/autoware_individual_params/tree/main/individual_params/config/default/aip_xx1
https://github.com/tier4/autoware_individual_params.jpntaxi/tree/main/individual_params/config/1/vehicle

Could you update individual_params too?

Reply:
Is it better to adjust structure of upstream individual_params to jpntaxi style?

Reply:
I think so.

Reply:
@tkimura4 please review https://github.com/tier4/autoware_individual_params/pull/24.

Reply:
@tkimura4 https://github.com/tier4/autoware_individual_params/pull/24 was merged.